### PR TITLE
Whats-New-5-28-in-context-alerting

### PR DESCRIPTION
**Pull request by daynaslord**
Photo - https://drive.google.com/file/d/15Vrje9FDl1Uf0EwOEtK9VHhI6q_4P7zD/view?usp=sharing
Release on 5/28
Ping directly for any questions or concerns

### Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### Are you making a change to site code?

If you're changing site code (rather than the content of a doc), please follow
[conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

### CODE REVIEW

**Comment by rhetoric101** [comment]
Reviewer: Please test this post, but do not merge. This is supposed to be published on 5/28/2021.

File: src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+summary: 'Now create alert conditions directly from the Query Builder'
+releaseDate: '2021-05-28'
+getStartedLink: 'https://one.newrelic.com/launcher/nr1-core.home?packages=nrai@0.2.417-c55ed87e&platform[timeRange][duration]=259200000&platform[$isFallbackTimeRange]=true&pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5ob21lLXNjcmVlbiJ9&overlay=eyJuZXJkbGV0SWQiOiJ3YW5kYS1kYXRhLWV4cGxvcmF0aW9uLmRhdGEtZXhwbG9yZXIiLCJpbml0aWFsQWN0aXZlSW50ZXJmYWNlIjoiY2hhcnRCdWlsZGVyIiwiaW5pdGlhbEFjY291bnRJZCI6MTYwNjg2MiwiaW5pdGlhbExpbWl0IjoxMCwiaW5pdGlhbFRpbWVXaW5kb3ciOnsic3RhcnQiOm51bGwsImVuZCI6Im5vdyIsIm9mZnNldCI6MTgwMDAwMH0sImluaXRpYWxUaW1lV2luZG93T3ZlcnJpZGUiOm51bGwsImluaXRpYWxGaWx0ZXJzIjoiIiwiaW5pdGlhbFBsb3RBZ2dyZWdhdGUiOnsicGxvdCI6bnVsbH0sImluaXRpYWxGYWNldCI6bnVsbCwiaW5pdGlhbENvbXBhcmVXaXRoIjpudWxsfQ=='
+---
+With the new ability to create an alert condition directly within the query builder, configuring alerts to cover all of your services and entities continues to get easier. If you are not comfortable with the New Relic Query Language (NRQL), you can create basic queries using the query builder tool, and then create an alert condition from the chart that you create.
```
Comment (urbiz-nr) [inline src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md:7]: you're

File: src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+---
+With the new ability to create an alert condition directly within the query builder, configuring alerts to cover all of your services and entities continues to get easier. If you are not comfortable with the New Relic Query Language (NRQL), you can create basic queries using the query builder tool, and then create an alert condition from the chart that you create. 
+
+The autocomplete functionality of the query builder lets you explore your data and create alerts without the need to know anything about NRQL. Alternatively, you can use the Data Explorer to browse the metrics and event data in your account, click your way to a chart, and create and create an alert condition from there as well. 
+
+When creating an alert, we recommend creating a dynamic threshold based alert, which we call a ‘baseline’ threshold, so that you do not need to understand the nuances of creating a static threshold configuration.  
+
+Below are the instructions to create an alert condition from query builder:
```
Comment (urbiz-nr) [inline src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md:13]: from the query builder

File: src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+---
+With the new ability to create an alert condition directly within the query builder, configuring alerts to cover all of your services and entities continues to get easier. If you are not comfortable with the New Relic Query Language (NRQL), you can create basic queries using the query builder tool, and then create an alert condition from the chart that you create. 
+
+The autocomplete functionality of the query builder lets you explore your data and create alerts without the need to know anything about NRQL. Alternatively, you can use the Data Explorer to browse the metrics and event data in your account, click your way to a chart, and create and create an alert condition from there as well.
```
Comment (urbiz-nr) [inline src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md:9]: I'd organize the sentences/paragraphs a little differently (see below).I'd also add links to the docs for the query builder and data explorer.

With the new ability to create an alert condition directly within the query builder, configuring alerts to cover all of your services and entities continues to get easier. 
If you are not comfortable with the New Relic Query Language (NRQL), you can create basic queries using the query builder tool, and then create an alert condition from the chart that you create. The autocomplete functionality of the query builder lets you explore your data and create alerts without the need to know anything about NRQL. 
Alternatively, you can use the Data Explorer to browse the metrics and event data in your account, click your way to a chart, and create and create an alert condition from there as well.

**Review by urbiz-nr — APPROVED** [review]
Nice feature! Left some suggestions, up to you!

File: src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md (hunk 3 above)
Comment (rhetoric101) [reply src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md:9]: @urbiz-nr , good suggestion! I'll do this!

File: src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md (hunk 2 above)
Comment (rhetoric101) [reply src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md:13]: @urbiz-nr, good catch!

File: src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md (hunk 1 above)
Comment (rhetoric101) [reply src/content/whats-new/2021/05/whats-new-5-28-in-context-alerting.md:7]: @urbiz-nr , thanks!

**Review by rhetoric101 — APPROVED** [review]
This post is ready to go!

**Pull request merged**